### DISCUSSION TITLE
Let instead of var

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#670] [(A) Anything]
Why cant we just use the let command instead of var

**Comment by PatheticMustan** [comment]
don't we?

**Reply by irhrhd** [reply]
I mean i think you guys do but some instances is says var

**Reply by irhrhd** [reply]
![A831EF49-FB62-4753-BC95-F0A0EFA088E0](https://user-images.githubusercontent.com/62729648/110681018-2d488a80-818e-11eb-8bf6-53d3fd20351c.png)
![D44DD45F-602D-4589-91F9-CE0C5750BD82](https://user-images.githubusercontent.com/62729648/110681036-32a5d500-818e-11eb-8279-ae96667f9570.png)
I actually dont think you guys do

**Comment by rusprice** [comment]
does it really matter? plus, let is only constant through it's respective block
